safe/metadata/provenance/provenance_step.py

```diff
@@ -54,7 +54,9 @@ def __str__(self):
         :return: the string representation
         :rtype: str
         """
-
+        # RMN:
+        # FIXME: Mismatched arguments. Should we delete the extra tags?
+        # pylint: disable=too-many-format-args
         return "%s: %s\n%s" % (
             self.time, self.title, self.description, self.data)
 
```

Comment:
We had a mismatch between placeholder and arguments. 3 placeholders and 4 args.
Should we delete one of the arg?